search-sentiments-map/src/main/webapp/script.js

```diff
@@ -24,9 +24,11 @@ let mapStyle = [{
   'stylers': [{'visibility': 'on'}, {'color': '#bfd4ff'}],
 }];
 let map;
-let dataMin = Number.MAX_VALUE;
-let dataMax = -Number.MAX_VALUE;
 let infowindow;
+// Hold the maximum and minium values of the sentiment scores. Set to these
+// as default to be replaced by any value.
```

Comment:
I meant for something like this :) 2 things:
1. Since you define min first, you want to write the 'minimum' in the comment first.
2. Say why you set these values.

```suggestion
// Hold the minimum and maximum values of the sentiment scores. By default, we set them to hold the maximum and minimum possible integer values respectively, so they can later be appropriately replaced by the actual sentiment scores.
```